src/RoleObjectFactory.js

```diff
@@ -164,12 +176,12 @@ function createAccessibilityObjectForRole(config) {
       accessibilityObject = new GroupData(displayObject, role, domIdPrefix);
       break;
 
-    case ROLES.HEADING1:
-    case ROLES.HEADING2:
-    case ROLES.HEADING3:
-    case ROLES.HEADING4:
-    case ROLES.HEADING5:
-    case ROLES.HEADING6:
+    case ROLES[`HEADING${1}`]:
+    case ROLES[`HEADING${2}`]:
+    case ROLES[`HEADING${3}`]:
+    case ROLES[`HEADING${4}`]:
+    case ROLES[`HEADING${5}`]:
+    case ROLES[`HEADING${6}`]:
```

Comment:
Why change it in this way?  For the previous version, I was just asking for the `eslint-disable-line` to include the particular linting rule being disabled.  While this should work the same as old case statements, it looks weird and I'm surprise that no-fallthrough isn't tripping up this approach as well.